src/fsharp/symbols/Symbols.fs

```diff
@@ -2090,8 +2100,23 @@ and FSharpMemberOrFunctionOrValue(cenv, d:FSharpMemberOrValData, item) =
                     let argtysl = m.GetParamTypes(cenv.amap, range0, m.FormalMethodInst) 
                     mkIteratedFunTy (List.map (mkRefTupledTy cenv.g) argtysl) rty
                 | V v -> v.TauType
-            NicePrint.prettyLayoutOfTypeNoCx (context.Contents cenv.g) ty
+            NicePrint.prettyLayoutOfTypeNoCx (displayContext.Contents cenv.g) ty
 
+    // TODO - consider the other types
```

Comment:
Probably not, might rename to `GetReturnTypeLayoutOfFunction` or something

Reply:
Since this is part of the FCS API I think it has to be `GetReturnTypeLayout` and be filled in for the other cases (or at least methods)  I can help do this if you like.  We generally avoid non-orthogonalities betweeen methods and functions in this API (which helps hide the somewhat artificial distinctions between these hanging around the compiler)